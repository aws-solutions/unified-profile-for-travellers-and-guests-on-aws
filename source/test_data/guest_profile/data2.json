{"modelVersion":"1.0","id":"0983908259","lastUpdatedOn":"2022-01-04T01:54:49.825486291Z","createdOn":"2021-11-04T01:54:49.825486291Z","lastUpdatedBy":"Cheyenne Yost","createdBy":"Ellis Bins","emails":[{"type":"business","address":"karolannglover@thompson.net","primary":true}],"phones":[{"type":"home","number":"(982)397-9544","primary":false,"countryCode":55},{"type":"mobile","number":"3563252173","primary":false,"countryCode":41}],"addresses":[{"type":"shipping","line1":"65889 Fork view","line2":"more content line 2","line3":"more content line 3","line4":"more content line 4","city":"Winston-Salem","state":{"code":"KY","name":"New Jersey"},"province":{"code":"","name":""},"postalCode":"98686","country":{"code":"CM","name":""},"primary":false},{"type":"","line1":"695 Ports port","line2":"more content line 2","line3":"more content line 3","line4":"more content line 4","city":"Santa Ana","state":{"code":"","name":""},"province":{"code":"ON","name":"Ontario"},"postalCode":"81057","country":{"code":"AW","name":""},"primary":false}],"honorific":"Mrs","firstName":"Norene","middleName":"Lawson","lastName":"Barton","gender":"female","pronoun":"she","dateOfBirth":"1938-02-20","language":{"code":"bg","name":""},"nationality":{"code":"TT","name":""},"jobTitle":"Supervisor","parentCompany":"Seabourne","loyaltyPrograms":[{"id":"4203161097","programName":"club","points":456337.0,"pointUnit":"miles","pointsToNextLevel":2882.0,"level":"silver","joined":"2021-11-04T01:54:49.825486291Z"},{"id":"4512771463","programName":"ascend","points":489696.0,"pointUnit":"reward points","pointsToNextLevel":214.0,"level":"platinium","joined":"2021-11-04T01:54:49.825486291Z"}]}