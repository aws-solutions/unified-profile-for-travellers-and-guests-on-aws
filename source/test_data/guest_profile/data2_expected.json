{"guest_profile_recs": [{"object_type": "pax_profile", "model_version": "1.0", "last_updated": "2022-11-04T02:26:33.862166999Z", "created_on": "2022-07-04T02:26:33.862166999Z", "last_updated_by": "Amparo Schmitt", "created_by": "Erick Harvey", "honorific": "Mx", "first_name": "Elliott", "middle_name": "Carleton", "last_name": "Daugherty", "gender": "other", "pronoun": "they", "date_of_birth": "2013-04-09", "job_title": "Administrator", "company": "US Green Data", "nationality_code": "AT", "nationality_name": "", "language_code": "fy", "language_name": "", "email_business": "", "email": "", "email_type": "", "phone": "6159425518", "phone_home": "", "phone_mobile": "", "phone_business": "", "phone_type": "personal", "address_line1": "9137 Corner shire", "address_line2": "more content line 2", "address_line3": "more content line 3", "address_line4": "more content line 4", "address_city": "Fresno", "address_state_province": "", "address_postal_code": "35992", "address_country": "CH", "address_billing_line1": "", "address_billing_line2": "", "address_billing_line3": "", "address_billing_line4": "", "address_billing_city": "", "address_billing_state_province": "", "address_billing_postal_code": "", "address_billing_country": "", "address_business_line1": "", "address_business_line2": "", "address_business_line3": "", "address_business_line4": "", "address_business_city": "", "address_business_state_province": "", "address_business_postal_code": "", "address_business_country": "", "address_mailing_line1": "", "address_mailing_line2": "", "address_mailing_line3": "", "address_mailing_line4": "", "address_mailing_city": "", "address_mailing_state_province": "", "address_mailing_postal_code": "", "address_mailing_country": "", "address_type": "", "traveller_id": "4105342576", "last_updated_partition": "2022/11/04/02/26/33/862166999Z"}], "common_email_recs": [], "common_phone_recs": [{"object_type": "phone_history", "model_version": "1.0", "last_updated": "2022-11-04T02:26:33.862166999Z", "last_updated_by": "Amparo Schmitt", "number": "6159425518", "country_code": 33, "type": "personal", "traveller_id": "4105342576", "last_updated_partition": "2022/11/04/02/26/33/862166999Z"}], "hotel_loyalty_recs": []}